### ISSUE TITLE
Implement tests for failover and "run only once" scenario

### ISSUE THREAD

**Issue by FlavioF** [FlavioF/quartz-scheduler-hazelcast-jobstore#3]
Implement tests to ensure that it is able to fail over between s1 and s2, so that
    A job runs only on s1 or s2
    If s1|s2 fails/crashes s2|s1 takes over

**Comment by FlavioF** [comment]
It is already covered in tests
